bin/requirements.yml

```diff
@@ -26,5 +26,8 @@
 - src: libre_ops.wal2json
   version: 1.0.5
 
+- src: libre_ops.multi_redis
```

Comment:
This got almost 50 downloads in the first 24 hours of going up on Ansible galaxy, so I guess people think it's useful :smile: